components/dot_menu/__snapshots__/dot_menu.test.tsx.snap

```diff
@@ -52,10 +52,11 @@ exports[`components/dot_menu/DotMenu should match snapshot, canDelete 1`] = `
   >
     <MenuItemAction
       onClick={[MockFunction]}
-      show={true}
+      show={false}
```

Comment:
Why is `show` changing here and in other places in the snapshot?

Reply:
done in latest commit